### ISSUE TITLE
RFC: Look into using S3 Standard-Infrequent Access (S3 Standard-IA) for backup 

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla-manager#3535]
Standard-IA cost less than S3 Standard:
* Standard-IA $0.0125 per GB
* S3 Standard $0.023 per GB - $0.021 per GB 
source: https://aws.amazon.com/s3/pricing/?nc=sn&loc=4

With a similar SLA. 

Backup are mostly write only.
Action items:
- Run dry run (even without the Manager!) of uploading backup to Standard-IA and validate the pricing / bandwidth and speed.
- Run a test for restore (can work slower, but still works)

**Comment by AccretiveUser** [comment]
Curious about your thoughts on this from that AWS link:

"S3 Standard-IA, and S3 One Zone-IA storage are charged for a minimum storage duration of 30 days, and objects deleted before 30 days incur a pro-rated charge equal to the storage charge for the remaining days. Objects that are deleted, overwritten, or transitioned to a different storage class before 30 days will incur the normal storage usage charge plus a pro-rated charge for the remainder of the 30-day minimum."

I had looked into possibly using Standard-IA for my backups, but because I rotate them daily and only keep the most recent 2, does this mean I would still get charged for 30 days worth of storage for any deleted objects?

**Comment by tzach** [comment]
Indeed, since we have a retention of 14 days of backup[1], Standard-IA looks less attractive.
Closing this issue for now, we can revisit it in the future.


[1] We could have an SSTable who lives much longer than 14 days

**Comment by mykaul** [comment]
> Indeed, since we have a retention of 14 days of backup[1], Standard-IA looks less attractive. Closing this issue for now, we can revisit it in the future.
> 
> [1] We could have an SSTable who lives much longer than 14 days

@tzach - I think we could extend the retention period to 30 days for the same cost as 14 days, if we move to IA.

**Comment by tzach** [comment]
I'm not aware of a customer request to extend the backup retention time
